src/Mumble.proto

```diff
@@ -20,6 +20,35 @@ message Version {
 	optional string os_version = 4;
 }
 
+// Send after the version packet, indicate the protocol preference of the client.
+// - This message is not required for text-only clients. It can be send before or after
```

Comment:
Do "text-only" clients even exist? :thinking: 

Besides: as we we want to use this for capabilities in general, I think even text-only clients might want to send this message at some point :shrug:

Reply:
Sure, as the capabilities of mumble extending...